src/Telegram.Bot/Types/MaskPosition.cs

```diff
@@ -12,21 +12,25 @@ public class MaskPosition
         /// <summary>
         /// The part of the face relative to which the mask should be placed. One of “forehead”, “eyes”, “mouth”, or “chin”.
         /// </summary>
+        [JsonProperty(Required = Required.Always)]
```

Comment:
How come it was fine before by having opt in and without this attribute on properties?

Reply:
Maybe there aren't anyone using it. I don't know. But it's definitely wouldn't work without either property name, naming strategy or contract resolver. I've tested it, json.net leaves fields as is (in pascal case).